dasharo-stability/capsule-update.robot

```diff
@@ -49,12 +49,15 @@ ${FUM_DIALOG_BOTTOM}=       The platform will automatically reboot and disable F
 *** Test Cases ***
 CUP001.001 Capsule Update With Wrong Keys
     [Documentation]    Check that DUT rejects flashing a capsule signed with invalid certificate.
+    Set Prompt For Terminal    Shell>
     Boot Into UEFI Shell
```

Comment:
Is there any meaningful difference between `Enter UEFI Shell` and `Boot Into UEFI Shell`

And again, it seems that setting of the prompt could take place in the keyword entering shell. We set e.g. the Linux prompt the same way, when logging into OS for the first time.

Reply:
I moved it to the Enter UEFI Shell kwd https://github.com/Dasharo/open-source-firmware-validation/commit/ea62cb11e4de15ab9f5da7e7cada863d73351048